compiler/transforms/convert_tosa_to_kernel.py

```diff
@@ -45,24 +46,13 @@ def match_and_rewrite(self, rescale_op: tosa.RescaleOp, rewriter: PatternRewrite
         # create linalg body with kernel op with the params of tosa ops
 
         # Extract all values:
-        input_zp = rescale_op.input_zp.value.data
-        assert_int8(input_zp)
-
-        output_zp = rescale_op.output_zp.value.data
-        assert_int8(output_zp)
-
+        input_zp = assert_int8(rescale_op.input_zp.value.data)
+        output_zp = assert_int8(rescale_op.output_zp.value.data)
         multiplier = rescale_op.multiplier.data.data[0].data
         assert isinstance(multiplier, int)
-
-        shift = rescale_op.shift.data.data[0].data
-        assert_int8(shift)
-
-        max_int = clamp_op.max_int.value.data
-        assert_int8(max_int)
-
-        min_int = clamp_op.min_int.value.data
-        assert_int8(min_int)
-
+        shift = assert_int8(rescale_op.shift.data.data[0].data)
+        max_int = assert_int8(clamp_op.max_int.value.data)
+        min_int = assert_int8(clamp_op.min_int.value.data)
```

Comment:
this makes pyright happy, to make sure that the values are integers

Reply:
Nice!